apps/condo/domains/billing/schema/RegisterBillingReceiptsService.js

```diff
@@ -74,7 +74,7 @@ const errors = {
 
 const getBillingPropertyKey = ({ address }) => address
 const getBillingAccountKey = ({ unitName, unitType, number, property }) => [unitName, unitType, number, getBillingPropertyKey(property)].join('_')
-const getBillingReceiptKey = ({ category: { id: categoryId }, period, property, account, recipient: { tin, bankAccount, iec, bic } }) => [categoryId, period, getBillingPropertyKey(property), getBillingAccountKey(account), tin, bankAccount, iec, bic].join('_')
+const getBillingReceiptKey = ({ category: { id: categoryId }, period, property, account, recipient: { tin, bankAccount, bic } }) => [categoryId, period, getBillingPropertyKey(property), getBillingAccountKey(account), tin, bankAccount, bic].join('_')
```

Comment:
How to build unique keys for objects:
```
billingPropertyKey = address
billingAccountKey = accountNumber + billingPropertyKey
billingReceiptKey = catrgoryId + period + billingAccountKey
```
How to calculate controlSum of billing receipt:
```
tin + bankAccount + bic + services
```